libass/ass.h

```diff
@@ -238,6 +238,16 @@ typedef enum {
      */
     ASS_FEATURE_BIDI_BRACKETS,
 
+    /**
+     * Allow breaking lines on all breakpoints considered legal by Unicode
+     * standard. It does not allow breaking words to avoid overfull lines.
+     * This is incompatible with VSFilter and disabled by default.
+     *
+     * This feature may be unavailable at runtime if libass was compiled without
+     * libunibreak support
```

Comment:
(I’d add a full stop at the end and rebalance the line break such that the two lines are of roughly equal width.)

Reply:
done.